Source/WebKit/Shared/WebEventConversion.cpp

```diff
@@ -38,6 +38,154 @@
 
 namespace WebKit {
 
+WebCore::MouseButton platform(WebMouseEventButton button)
+{
+    switch (button) {
+    case WebMouseEventButton::None:
+        return WebCore::MouseButton::None;
+    case WebMouseEventButton::Left:
+        return WebCore::MouseButton::Left;
+    case WebMouseEventButton::Middle:
+        return WebCore::MouseButton::Middle;
+    case WebMouseEventButton::Right:
+        return WebCore::MouseButton::Right;
+    default:
+        RELEASE_ASSERT_NOT_REACHED();
+    }
+}
+
+WebMouseEventButton kit(WebCore::MouseButton button)
+{
+    switch (button) {
+    case WebCore::MouseButton::None:
+        return WebMouseEventButton::None;
+    case WebCore::MouseButton::Left:
+        return WebMouseEventButton::Left;
+    case WebCore::MouseButton::Middle:
+        return WebMouseEventButton::Middle;
+    case WebCore::MouseButton::Right:
+        return WebMouseEventButton::Right;
+    default:
+        RELEASE_ASSERT_NOT_REACHED();
+    }
+}
+
+WebCore::PlatformEvent::Type platform(WebEventType type)
+{
+    switch (type) {
+    // Mouse
+    case WebEventType::MouseDown:
+        return WebCore::PlatformEvent::Type::MousePressed;
+    case WebEventType::MouseUp:
+        return WebCore::PlatformEvent::Type::MouseReleased;
+    case WebEventType::MouseMove:
+        return WebCore::PlatformEvent::Type::MouseMoved;
+    case WebEventType::MouseForceChanged:
+        return WebCore::PlatformEvent::Type::MouseForceChanged;
+    case WebEventType::MouseForceDown:
+        return WebCore::PlatformEvent::Type::MouseForceDown;
+    case WebEventType::MouseForceUp:
+        return WebCore::PlatformEvent::Type::MouseForceUp;
+
+    // Wheel
+    case WebEventType::Wheel:
+        return WebCore::PlatformEvent::Type::Wheel;
+
+    // Keyboard
+    case WebEventType::KeyDown:
+        return WebCore::PlatformEvent::Type::KeyDown;
+    case WebEventType::KeyUp:
+        return WebCore::PlatformEvent::Type::KeyUp;
+    case WebEventType::RawKeyDown:
+        return WebCore::PlatformEvent::Type::RawKeyDown;
+    case WebEventType::Char:
+        return WebCore::PlatformEvent::Type::Char;
+
+#if ENABLE(TOUCH_EVENTS)
+    // Touch
+    case WebEventType::TouchStart:
+        return WebCore::PlatformEvent::Type::TouchStart;
+    case WebEventType::TouchMove:
+        return WebCore::PlatformEvent::Type::TouchMove;
+    case WebEventType::TouchEnd:
+        return WebCore::PlatformEvent::Type::TouchEnd;
+    case WebEventType::TouchCancel:
+        return WebCore::PlatformEvent::Type::TouchCancel;
+#endif
+
+#if ENABLE(MAC_GESTURE_EVENTS)
+    // Gesture
+    case WebEventType::GestureStart:
+        return WebCore::PlatformEvent::Type::GestureStart;
+    case WebEventType::GestureChange:
+        return WebCore::PlatformEvent::Type::GestureChange;
+    case WebEventType::GestureEnd:
+        return WebCore::PlatformEvent::Type::GestureEnd;
+#endif
+
+    default:
+        RELEASE_ASSERT_NOT_REACHED();
```

Comment:
Maybe just keep `ASSERT_NOT_REACHED()`, like before? Seems harsh to crash here

Reply:
Having said that we were inconsistent before too, I see at least one `RELEASE_ASSERT_NOT_REACHED()` in the deleted lines.